src/main/java/FilterServlet.java

```diff
@@ -0,0 +1,23 @@
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+
+public class FilterServlet implements Filter {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All classes, interfaces, enums and annotations must belong to a named package](https://app.codacy.com/manual/thejerome/IFMO_Servlets_20190915/pullRequest?prid=4840007)